bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -270,7 +271,7 @@ class ItcImagingTable(val parameters: ItcParametersProvider) extends ItcTable {
         case INSTRUMENT_GSAOI  =>
           new ItcGsaoiImagingTableModel(keys, uniqueConfigs, inputs, results)
 
-        case INSTRUMENT_GNIRS | INSTRUMENT_NIFS | INSTRUMENT_NIRI =>
+        case INSTRUMENT_NIRI =>
```

Comment:
With this change the `showCoadds` parameter value changes from `true` to `false` for NIFS as well as GNIRS.  I imagine that is what you intended but I thought I'd check since this change was mostly about GNIRS.